pkg/tcl/apitcl/v1/testworkflows.go

```diff
@@ -82,23 +83,34 @@ func (s *apiTCL) DeleteTestWorkflowsHandler() fiber.Handler {
 	errPrefix := "failed to delete test workflows"
 	return func(c *fiber.Ctx) error {
 		selector := c.Query("selector")
-		_, err := s.TestWorkflowsClient.List(selector)
+		workflows, err := s.TestWorkflowsClient.List(selector)
 		if err != nil {
 			return s.BadGateway(c, errPrefix, "client problem", err)
 		}
 
+		// Delete
 		err = s.TestWorkflowsClient.DeleteByLabels(selector)
 		if err != nil {
 			return s.ClientError(c, errPrefix, err)
 		}
 
-		//skipExecutions := c.Query("skipDeleteExecutions", "")
-		//for range workflows.Items {
-		//	s.Metrics.IncDeleteTestWorkflow(err)
-		//	if skipExecutions != "true" {
-		//		// TODO: Delete Executions
-		//	}
-		//}
+		// Mark as deleted
+		for range workflows.Items {
+			s.Metrics.IncDeleteTestWorkflow(err)
+		}
+
+		// Delete the executions
+		skipExecutions := c.Query("skipDeleteExecutions", "")
+		if skipExecutions != "true" {
```

Comment:
should we check for bool?

Reply:
What do you mean?

Reply:
`ParseBool` accepts multiple values as truthy - not sure if not better to be consistent with `true` only (like [**here**](https://github.com/kubeshop/testkube/blob/develop/internal/app/api/v1/tests.go#L505-L511))

Reply:
ok, let's follow existng approach